test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -43,10 +43,10 @@ function parent() {
   child.stdout.on('data', function(c) {
```

Comment:
How about adding a `mustCallAtLeast`?
(I see that the value of `n` is asserted, but this will give a one-step-closer-to-root-cause assertion error)